kiwmi/desktop/view.c

```diff
@@ -247,29 +189,37 @@ view_resize(struct kiwmi_view *view, uint32_t edges)
     view_begin_interactive(view, KIWMI_CURSOR_RESIZE, edges);
 }
 
-/**
- * Creates a kiwmi_view_child for each subsurface of either the 'child' or the
- * 'view'. 'child' can be NULL, 'view' should never be.
- */
-void
-view_init_subsurfaces(struct kiwmi_view_child *child, struct kiwmi_view *view)
+static struct kiwmi_output *
+view_toplevel_get_output(struct kiwmi_toplevel *toplevel)
```

Comment:
Kinda self-note: should try using `wlr_scene_surface::primary_output`, though that may be somewhat hacky as we don’t have direct access to the `wlr_scene_surface` (we could also use an algorithm similar to theirs instead).

Reply:
Not gonna do this, actually.